io.mrarm.mcpelauncher.metainfo.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>io.mrarm.mcpelauncher</id>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>GPL-3.0</project_license>
+    <name>Minecraft Bedrock Launcher</name>
+    <summary>Download and play Minecraft Bedrock Edition on GNU/Linux</summary>
+
+    <description>
+        <p>
+            The project runs the native libraries from the Android version directly on your computer. This is accomplished by fixing the incompatibilities between the libc used on Android and the one used on desktop Linux or OS X (Android - Bionic; Linux - glibc). This is a simple compatibility layer which doesn’t impact performance in any significant way; it’s similar to Wine, but much more lightweight and simple. Aditionally, all Android-specific code has been rewritten to run on Linux (AppPlatform, Store, Xbox Live, etc.).
```

Comment:
Please add disclaimer like the one in the app that it's not unofficial and not affiliated with MS/Mojang. For example: https://github.com/flathub/com.slack.Slack/blob/master/com.slack.Slack.appdata.xml#L18

Reply:
I have added the discaminar from the launcher welcome page to the metadata